docker/sentinel-compose.yml

```
@@ -0,0 +1,91 @@
+    
+  
+  redis-server-2:
+    image: falkordb/falkordb:latest
+    hostname: redis-server-2
+    container_name: redis-server-2
+    ports:
+      - 6381:6381
+    networks:
+      - redis-sentinel
+    environment:
+      REDIS_ARGS:  "--port 6381 --replica-announce-ip redis-server-1 --replica-announce-port 6381 --replicaof redis-server-1 6380"
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)